cogs/mod.py

```diff
@@ -45,6 +47,37 @@ def __init__(self, bot):
             await ctx.send(default.actionmessage("kicked"))
         except Exception as e:
             await ctx.send(e)
+            
+    @commands.has_permissions(administrator=True)
+    @commands.command()

```

Comment:
This is not a command that I will be adding.
1. No [at]everyone protection
2. Easy to make on your own